docs/CHANGELOG.md

```diff
@@ -16,56 +19,87 @@ Previously, the Checker Framework interpreted
 
 No code changes are required for type systems that default explicit and implicit
-bounds the same, such as the Locking Checker.
-
-Here is how to find places that your code might need to be changed.  Call a
-"non-top-default type system" one of the ones at listed at
-https://checkerframework.org/manual/#default-is-not-top-type-systems.  Consider:
- * every occurrence of "`extends Object`" without an annotation:
-    * that is in a file being typechecked by a non-default-top type system, or
-    * that contains `@AnnotatedFor` for a non-default-top type system (this is used
-      in annotated libraries).
- * every occurrence of `@Anno T extends Object` or `T extends @Anno Object`, where
-   `@Anno` is an annotation from a non-top-default type system.
+bounds the same, such as the Locking Checker, even though its default type
+qualifer is not the top type qualifier.
+
+If the default type qualifier differs from the top type qualifier, you need to
+update some code and annotated libraries, namely
+
+ * every file that is typechecked by a non-default-top type system, and
+ * every file that contains `@AnnotatedFor` for a non-default-top type system
+   (this is used in annotated libraries).
+
+Make the following changes:
+
+[//]: # (Comment: GitHub Markdown does not render the italics, but other Markdown engines do.)
+
+ * change "`<... extends Object>`" to "<code>&lt;... extends @<em>DefaultType</em> Object&gt;</code>".
+   If your code contains "`<... extends Object>`" where arbitrary instantiation is
+   desirable, then your previous annotations were buggy and should have been
+   "<code>&lt;... extends @<em>TopType</em> Object&gt;</code>",
+   but that can be written as just "`<...>`", which is better style.
+ * optionally, remove <code>extends @<em>TopType</em> Object&gt;</code>.
+   This change is not required, but it is recommended as a matter of style.
+   (If the code is annotated for two type systems, then change
+   "<code>&lt;T extends @<em>TopType1 TopType2</em> Object&gt;</code>" to "`<T>`",
+   but leave "<code>&lt;T extends @<em>TopType1 NotTopType2</em> Object&gt;</code>"
+   unchanged.)
+
+For wildcards, the same transformations apply, with one exception.  "`<@Anno ?
+extends @Anno Object>`" can be abbreviated "`<@Anno ?>`", but the same
+abbreviation does not apply to non-wildcard type variables.  That is, "`<@Anno T
+extends @Anno Object>`" cannot be abbreviated unless `@Anno` is the top type, in
+which case it is equivalent to just "`<T>`".
+
+
+As an example, for the Nullness Checker,
+
+ * `<T extends Object>`: change to `<T extends @NonNull Object>`
+ * `<T extends @Nullable Object>`: change to `<T>` for better style
+ * `<T extends @NonNull Object>`: don't change
+ * `<T>`: don't change
+
+ * `<@Nullable T extends Object>`: did not make sense (and has a different meaning now)
+ * `<@Nullable T extends @Nullable Object>`: change to `<T>` for better style
```

Comment:
This is wrong.  `<T>` is `<@NonNull T extends @Nullable Object>`.
`<@Nullable T extends @Nullable Object>` can be written as just `<@Nullable T>`